frontend/packages/schema-editor/src/components/SchemaInspector/SchemaInspector.tsx

```diff
@@ -17,10 +17,9 @@ export const SchemaInspector = () => {
 
   if (!selectedUniquePointer) {
     return (
-      <div>
-        <p className={classes.noItem}>{t('schema_editor.no_item_selected')}</p>
-        <Divider />
-      </div>
+      <StudioLabelAsParagraph size='sm' className={classes.noItem}>
+        {t('schema_editor.no_item_selected')}
+      </StudioLabelAsParagraph>
```

Comment:
Maybe this could be moved to a separate file so we dont need to duplicate the `noItem` css in two files? 😊  
Then we can use it both here and in `TypeInspector`

Reply:
Done 😄